### PR TITLE
#85 add expander

### LINKED ISSUE #85: Add `Expander` widget
https://docs.gtk.org/gtk4/class.Expander.html

### PR DESCRIPTION
**Pull request by PhilippMDoerner**
Closes #85 

Adds the expander widget as described.

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -167,6 +167,10 @@ The `widgets` directory contains examples for how to use different widgets.
     <td><a href="https://github.com/can-lehmann/owlkettle/blob/main/examples/widgets/text_view.nim">Text View</a></td>
     <td><img alt="Text View" src="../docs/assets/examples/text_view.png" width="757px"></td>
   </tr>
+  <tr>
+    <td><a href="https://github.com/can-lehmann/owlkettle/blob/main/examples/widgets/expander.nim">Expander</a></td>
+    <td><img alt="Expander" src="../docs/assets/examples/expander.png" width="922px"></td>
+  </tr>
```
Comment (can-lehmann) [inline examples/README.md:173]: The list should be sorted alphabetically

File: owlkettle/widgets.nim [hunk 2]
```diff
@@ -3525,6 +3525,79 @@ renderable Scale of BaseWidget:
         echo "New value is ", newValue
         app.value = newValue
 
+renderable Expander of BaseWidget:
+  ## Wrapper for GTK Expander Widget: https://docs.gtk.org/gtk4/class.Expander.html
```
Comment (can-lehmann) [inline owlkettle/widgets.nim:3529]: Usually this is a summary of what the widget does & does not include a link to the GTK docs.
```suggestion
  ## Container that shows or hides its child depending on whether it is expanded/collapsed 
```

**Review by can-lehmann — COMMENTED** [review]
Already looks really good so far.

File: owlkettle/widgets.nim (hunk 2 above)
Comment (PhilippMDoerner) [reply owlkettle/widgets.nim]: Fair. I had written that one before I saw you preferred not to have direct gtk doc links.
To me it just seemed sensible that users should have a link to the more expansive docs for details about the Widget, thus I had included them there. 
Spare them the necessity of a google search where they might have bad googling skills and not find the right docs and all that.

**Comment by can-lehmann** [comment]
Docs need to be rebuilt, then this should be ready to merge.

**Comment by PhilippMDoerner** [comment]
Aaaaand done

**Comment by PhilippMDoerner** [comment]
Wait, no crap, god damnit I was just merge main back into things >_<

**Comment by PhilippMDoerner** [comment]
There we go

**Comment by can-lehmann** [comment]
Looks good!

**Pull request merged**